crates/bevy_ui/src/geometry.rs

```diff
@@ -459,11 +457,10 @@ impl UiRect {
     /// assert_eq!(ui_rect.top, Val::ZERO);
     /// assert_eq!(ui_rect.bottom, Val::ZERO);
     /// ```
-    pub fn left(value: Val) -> Self {
-        UiRect {
-            left: value,
-            ..Default::default()
-        }
+    pub const fn left(value: Val) -> Self {
+        let mut rect = UiRect::DEFAULT;
+        rect.left = value;
+        rect
     }
 
```

Comment:
I'd prefer that we used struct update syntax for the constructor functions, like:

```suggestion
    pub const fn left(left: Val) -> Self {
        Self {
            left,
            ..Self::DEFAULT
        }
    }

```

is simpler and clearer.

Reply:
I thought you can, can't you? `Self::DEFAULT` is a const value, it's not `Self::default()`